src/precice/types.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+namespace precice {
+
+/**
+ * Type used for the IDs of vertices
+ */
+using VertexID = int;
+
+/**
+ * Type used for the IDs of edges
+ */
+using EdgeID = int;
+
+/**
+ * Type used for the IDs of triangles
+ */
+using TriangleID = int;
+
+/**
+ * Type used for the IDs of matching entities
+ */
+using MatchID = int;
```

Comment:
In my PR, @uekerman requested some documentation clarification on this.

Reply:
Used in nearest projection and could really go into the query package.

A nearest projection can match vertices, edge and triangles, so the type of the match is unclear.